docs/document/content/reference/distsql/syntax/rdl/add-resource.en.md

```diff
@@ -0,0 +1,97 @@
++++
+title = "RDL Syntax"
+weight = 2
++++
+
+
+## ADD RESOURCE
+
+The `ADD RESOURCE` syntax is used to add resources for the currently selected schema.
+
+
+
+### Syntax
+```SQL
+AddResource ::=
+  'ADD' 'RESOURCE' dataSource (',' dataSource)*
+
+dataSource ::=
+  dataSourceName '(' ( 'HOST' '=' hostName ',' 'PORT' '=' port ',' 'DB' '=' dbName  |  'URL' '=' url  ) ',' 'USER' '=' user (',' 'PASSWORD' '=' password )?  (',' 'PROPERTIES'  '(' ( key  '=' value ) ( ',' key  '=' value )* ')'  )?')'
+
+dataSourceName ::=
+  identifier
+
+hostname ::=
+  identifier | ip
+
+dbName ::=
+  identifier
+
+port ::=
+  int
+
+password ::=
+  identifier | int | string 
+
+user ::=
+  identifier
+
+url ::=
+  identifier | string
+
+```
+
+ ### Supplement
+- Before adding resources, please confirm that a schema has been created in Proxy, and execute the `use` command to successfully select a schema
+- Confirm that the added resource can be connected normally, otherwise it will not be added successfully
+- `dataSourceName` is case-sensitive
+- `dataSourceName` needs to be unique within the current schema
+- `dataSourceName` name only allows letters, numbers and `_`, and must start with a letter
+- `poolProperty` is used to customize connection pool parameters, `key` must be the same as the connection pool parameter name, `value` supports int and String types
+- String mode is recommended when `password` contains special characters
+
+ ### Example
+- Add resources using standard mode
+```SQL
+ADD RESOURCE ds_0 (
+    HOST=127.0.0.1,
+    PORT=3306,
+    DB=db_0,
+    USER=root,
+    PASSWORD=root
+);
+```
+
+- Add resources and set connection pool parameters using standard mode
```

Comment:
Hi @lanchengx  Because there is only one resource in code example, I think `Add resource` is enough.